default/scripting/techs/defense/mines.macros

```diff
@@ -80,7 +80,7 @@ EG_SYSTEM_MINES
                     Not HasTag name = "UNOWNED_FRIENDLY"
                     ]
                 ]
-                Structure low = @1@ + 0.0001
+                Structure high = @1@ + 0.0001
```

Comment:
seems to me that this line should be the same as line 63 in the empire-owned planet case,
``` Structure high = @1@```